sergeev_fedor/graph_generator.cpp

```diff
@@ -0,0 +1,79 @@
+#include <iostream>
+#include <vector>
+
+constexpr int kVerticesCount = 14;
+
+class Graph {
+ public:
+  using VertexId = int;
+  using EdgeId = int;
+
+  void add_vertex() {
+    vertices.emplace(vertices.end(), (Vertex(vertex_id_++)));
```

Comment:
1) Проще использовать `emplace_back`
```suggestion
    vertices.emplace_back(vertex_id_++);
```
2) Лучше выделить логику по генерации нового уникального id в отдельную функцию, например
```cpp
VertexId get_new_vertex_id() {
  return vertex_id_++;
}
```